egs/yomdle_tamil/v1/local/chain/run_e2e_cnn.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
```

Comment:
Would it be possible to add some results to these scripts?

Reply:
Ok, thank you, I will add it.